config/initializers/_config.rb

```diff
@@ -7,6 +7,7 @@ module Convection
 Convection.config =
   OpenStruct.new(
     access_token: ENV['ACCESS_TOKEN'] || 'replace-me',
+    admin_names: (ENV['ADMIN_NAMES'] || 'Alice Betty Cindy').split,
```

Comment:
I see 😄

Reply:
It's worth noting that we have stitching in Gravity: https://github.com/artsy/metaphysics/tree/master/src/lib/stitching/gravity